CHANGES/7781.bugfix

```diff
@@ -0,0 +1,2 @@
+Taught pre-migration to order content by last-updated, in order to be able to reliably
+recover from errors during migration attempts.
```

Comment:
Do you think it's a bad idea if we also mention here that Pulp 2.21.5+ is required from now on? 
I understand that for ultra small setups it won't be the case. However you mentioned that it fails even with one RHEL repo, correct? So threshold is extremely low :/

Reply:
Fine idea - change incoming.

Reply:
Updated